tests/model_creators/specific_creators.py

```diff
@@ -241,7 +241,7 @@ def create_stock_from_event_occurrence(event_occurrence: Dict,
 
 
 def create_stock_from_offer(offer: Offer,
-                            price: int = 10,
+                            price: float = 10,
```

Comment:
Cool bien vu !
```suggestion
                            price: float = 10,2,
```
Histoire d'avoir une signature plus réelle ?

Reply:
Yes ! 9.90 c'est plus réel :) (et pour info c'est un '.' et pas une ',' le séparateur des float en python)